pkg/backup_servers/sftp_server.go

```diff
@@ -0,0 +1,162 @@
+//
+// Copyright (c) 2021 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+package backup_servers
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"strings"
+
+	orgv1 "github.com/eclipse-che/che-operator/pkg/apis/org/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// SftpServer implements BackupServer
+type SftpServer struct {
+	config orgv1.SftpServerConfing
+	ResticClient
+}
+
+func (s *SftpServer) PrepareConfiguration(client client.Client, namespace string) (bool, error) {
```

Comment:
```suggestion
func (s *SftpServer) PrepareConfiguration(client client.Client, namespace string) (done bool, err error) {
```
it would help to understand what is `return true, fmt.Errorf("SFTP server username must be configured")` and similar

Reply:
Thamk you for the remark. However, by default, if a function returns `(bool, error)` this means it returns done status and error if any. This is a convention for the backup and restore controllers. In case when a function returns something else or have, say, another return sequence/parameters, it is always described in a comment or variable name.